dist/deserializers/params-fields.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
```

Comment:
`dist` repository must be "git ignored".

Reply:
If we add dist to gitignore, we then can't add the package using the git url. Both appoach are common, but if you prefer to not have an automatic commit for the transpioled code, I can do it.

Reply:
Here is what I read about having an NPM package using Babel: http://2ality.com/2017/07/npm-packages-via-babel.html. Looks simple and the guy does not version the generated code.